db/migrate/20210115103021_add_subscriber_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddSubscriberToUser < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :subscriber, :boolean, default: false
```

Comment:
Do we need new column? I thought we didn't because we have enum for role!🤔

Reply:
we wanted to set the default to false. but it is now changed and it works :)